CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,81 @@
+# PureScript Community Code of Conduct
+
+The PureScript community follows some simple ground rules to protect safe, inclusive spaces to discuss the PureScript programming language.
```

Comment:
Should the comma between "safe" and "inclusive" be replaced with an "and"? And perhaps replace "to" with "where we can". So would read "The PureScript community follows some simple ground rules to protect safe and inclusive spaces where we can discuss the PureScript programming language."

Reply:
I think either rendering (Thomas' current one and your suggested one) still say the same thing. So, I don't really have an opinion here.